### ISSUE TITLE
iData_vDOS: seems something is broken

### ISSUE THREAD

**Issue by farhi** [McStasMcXtrace/iFit#179]
Issue:

- Sqw=sqw_cubic_monoatomic; D=dos(Sqw);

produces D as a scalar ...

- s = sqw_phonons('POSCAR_Al', 'emt'); D=dos(s);

brings error:

```
??? Error using ==> iData_vDOS.iData_vDOS>iData_vDOS.iData_vDOS at 92
iData_vDOS: the given input iData does not seem to be convertible to iData_vDOS.

Error in ==> iFunc_Sqw4D.dos at 100
    DOS = iData_vDOS(DOS);
```

**Comment by farhi** [comment]
This will mostly be solved when implementing #181

**Comment by farhi** [comment]
Was functional for release 2.0.2 at commit https://github.com/McStasMcXtrace/iFit/commit/d703eed4ac8d82956fb8adf480a90716df192dc0
